__tests__/app.test.js

```diff
@@ -132,3 +133,52 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with an array of comment objects for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(Array.isArray(comments)).toBe(true);
+        expect(comments.length).toBe(11);
+      });
+  });
+
+  test("status: 200 each comment has the properties which match comment columns", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect(comment).toMatchObject({
+            article_id: expect.any(Number),
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+          });
+        });
+      });
+  });
+});
+
+test("400: responds with an error when article_id is not an integer", () => {
+  return request(app)
+    .get("/api/articles/first_article/comments")
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Bad request");
+    });
+});
+
+test("404: responds with an error when given an article_id that doesn't exist", () => {
+  return request(app)
+    .get("/api/articles/9999/comments")
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Article not found");
+    });
+});
```

Comment:
Great tests so far and error testing but there's one more scenario we need to test here:
We need to check that if an article exists but has no comments, when we request its comments, it should return and empty array (and not result in an error)